src/renderer/pages/Dashboard.vue

```diff
@@ -71,8 +75,17 @@ export default {
     WalletButtonImport
   },
 
+  provide () {
+    // The `MarketChartHeader` component would use these methods when wrapped by `MarketChart`
+    return {
+      changePeriod () {},
```

Comment:
Move this to the `MarketChartHeader`, the [inject api](https://vuejs.org/v2/api/#provide-inject) supports default params.